Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -114,6 +115,18 @@ - (id)init:(std::shared_ptr<Babylon::Plugins::Camera::Impl::ImplData>)implData;
                 ];
             }
             
+            // In portrait mode swap height and width when selecting the best format.
+            uint32_t targetFormatHeight{maxHeight};
+            uint32_t targetFormatWidth{maxWidth};
+            
+            if (m_implData->cameraTextureDelegate->videoOrientation == AVCaptureVideoOrientationPortrait
+                ||  m_implData->cameraTextureDelegate->videoOrientation == AVCaptureVideoOrientationPortraitUpsideDown)
+            {
+                targetFormatHeight = maxWidth;
+                targetFormatWidth = maxHeight;
```

Comment:
Any reason not to just swap maxWidth and maxHeight rather than introduce a new set of width/height vars?

Reply:
You can't make parameters __block variables in Objective C when marshalling to another thread, so they become immutable :/

Reply:
Reverted this change, as the web API is actually already expecting width and height to correspond to the sensor oriented dimensions rather than the device oriented dimensions.